package.json

```diff
@@ -91,6 +95,11 @@
     "pretest": "npm run build",
     "test": "npm run lint && karma start",
     "release": "./scripts/release.sh",
-    "start": "grunt"
+    "serve": "./scripts/open.js && parallelshell 'npm run watch' 'http-server -p 5678'",
```

Comment:
You can actually make this a lot simpler. `npm run watch & http-server -p 5678 -c-1 -o`

`-c-1` turns off caching. `-o` tells `http-server` to just open up the root of the web server which I think is fine I would rather have less dependencies.

You also don't need `parallelshell`. If you need to run multiple commands at once. Just separate them with `&` it works on both Mac and Windows.